content/applications/productivity/mail_plugins/gmail.rst

```diff
@@ -2,94 +2,143 @@
+
+Delete the following three lines of text from the :guilabel:`login` TS file:
+
+.. code-block:: python
+
+   if (!/^https:\/\/([^\/?]*\.)?odoo\.com(\/|$)/.test(validatedUrl)) {
+        return notify("The URL must be a subdomain of odoo.com");
+   }
+
+This removes the odoo.com domain constraint from the Gmail Plugin program.
+
+Next, in the ZIP file, go to :menuselection:`mail-client-extensions-master --> gmail`, and open the
+file called :guilabel:`README`. Follow the instructions in the :guilabel:`README` file to push the
+Gmail Plugin files as a Google Project.
+
+.. note::
+   Your computer must be able to run Linux commands in order to follow the instructions on the :guilabel:`README` file.
+
+Share the Google Project with the Gmail account that you wish to connect with Odoo. Then click
+:guilabel:`Publish` and :guilabel:`Deploy from manifest`. Lastly, click
+:guilabel:`Install the add-on` to install the Gmail Plugin.
+
+Configure your Odoo database
+----------------------------
 
-#. The right-side panel can now display **Company Insights**. At the bottom, click on *Login*.
+The :guilabel:`Mail Plugin` feature must be enabled in your Odoo database in order to use the Gmail
+Plugin. Go to :menuselection:`Settings --> General Settings`. Under :guilabel:`Integrations`,
+activate :guilabel:`Mail Plugin`, and then click :guilabel:`Save`.
 
-   .. image:: gmail/panel-login.png
-      :align: center
-      :alt: Logging in your Odoo database
+.. image:: gmail/mail-plugin-setting.png
+   :align: center
+   :alt: Mail Plugin feature in Settings
 
-   .. note::
-      Only a limited amount of *Company Insights* (*Lead Enrichment*) requests are available as a
-      trial. This feature requires :ref:`prepaid credits <mail_plugins/pricing>`.
+Configure your Gmail inbox
+--------------------------
 
-#. Enter your Odoo database URL and click on *Login*.
+In your Gmail inbox, a purple Odoo icon will now appear on the right sidebar. Click on the Odoo
+icon to open up the Odoo plugin window. Then, click on any email in your inbox. Click
+:guilabel:`Authorize Access` in the plugin window to grant Odoo access to your Gmail inbox.
 
-   .. image:: gmail/database-url.png
-      :align: center
-      :alt: Entering your Odoo database URL
+.. image:: gmail/authorize-access.png
+   :align: center
+   :alt: Authorize Access button in the right sidebar of the Odoo plugin panel
 
-#. If you aren't logged into your database, enter your credentials.
+Next, click :guilabel:`Login`. Enter the URL of the Odoo database that you wish to connect with
+your Gmail inbox, and log in to the database.
 
-#. Click on *Allow* to let the Gmail Plugin connect to your database.
+.. note::
+   Use the general URL for the database, not the URL of a specific page in the database. For example, use `https://stealthywood.odoo.com`, not `https://stealthywood.odoo.com/web#cids=1&action=menu`.
```

Comment:
```suggestion
   Use the general URL for the database, not the URL of a specific page in the database. For example,
   use `https://mycompany.odoo.com`, not `https://mycompany.odoo.com/web#cids=1&action=menu`.
```

Also, make sure the first letter of the second line of the note lines up with the first letter of the first line!